common_test.go

```diff
@@ -217,23 +249,27 @@ func benchmarkRandomReadsWrites(b *testing.B, db DB) {
 	for i := 0; i < b.N; i++ {
 		// Write something
 		{
-			idx := int64(rand.Int()) % numItems // nolint:gosec testing file, so accepting weak random number generator
+			idx := rand.Int63n(numItems) // nolint:gosec testing file, so accepting weak random number generator
 			internal[idx]++
 			val := internal[idx]
 			idxBytes := int642Bytes(idx)
 			valBytes := int642Bytes(val)
 			//fmt.Printf("Set %X -> %X\n", idxBytes, valBytes)
 			err := db.Set(idxBytes, valBytes)
-			b.Error(err)
+			if err != nil {
```

Comment:
not sure why you need `if`

Reply:
Performance. `require.NoError()` is very costly, so without the `if` the benchmark time is dominated by `require.NoError()` (profiler said about 60%, if I remember correctly).

Reply:
kk. then why not use if err != nil { b.Fatal(err) } ?

Reply:
Because require.NoError() generates nice output automatically. :) But sure, I'll update.